src/qibolab/platforms/abstract.py

```diff
@@ -270,23 +269,21 @@ def transpile(self, circuit: Circuit):
         return sequence
 
     @abstractmethod
-    def execute_pulse_sequence(self, sequence, nshots=1024, relaxation_time=None, raw_adc=False):
+    def execute_pulse_sequence(self, sequence, nshots=1024, relaxation_time=None):
```

Comment:
Here you should keep also `raw_adc`. Probably something went wrong when you merged main.

Reply:
This is important to fix before merging because it will break other features.